src/components/app/index.tsx

```diff
@@ -14,7 +14,7 @@ import Router from '../router';
 
 export default hot(function App() {
   return (
-    <ThemeProvider theme={theme}>
+    <ThemeProvider theme={theme} useGlobalStyles={false}>
```

Comment:
Her vil vi ta i bruk globale stiler - applikasjonen er lagt opp for det. Det er da greit å rydde opp i main entrypoint index.html for å ta bort link til Heebo font og slette reset og normalise stiler definert i appen.

Reply:
Bruker global stil, men i tillegg bruke global stil fra repo siden den er custom